### ISSUE TITLE
Property based testing

### ISSUE THREAD

**Issue by dbrgn** [saltyrtc/saltyrtc-server-python#14]
Just found this:

http://hypothesis.works/articles/getting-started-with-hypothesis/

Looks interesting, maybe it would be something for the server?

**Comment by lgrahl** [comment]
Two years later... :laughing: 

Fuzzing is nice for testing purposes. But for automatic tests, it's not as useful since it can result in random failures (due to its randomness). I think type hints (#88) will take care of the major part of input validation. And if someone is using a different type, I'm absolutely fine with weird behaviour. Last but not least, the server doesn't have a large API surface.

Anyway, TL;DR, I think this would require too much effort for little effect, at least in this project. (Fuzzing would be extremely useful for libraries such as MsgPack.)
